.github/workflows/ci.yml

```diff
@@ -358,7 +358,7 @@ jobs:
           ls -lhR test-results
           find test-results -mindepth 1 -name '*.json' -mtime +7 -delete
   test-collect-coverage:
-    needs: [unit-test, integration-test, acceptance-test]
+    needs: [unit-test, integration-test]
```

Comment:
Are the workflows and `go-test.py` changes meant to be merged as well?

Reply:
No. They're to illustrate coverage changes.